### PR TITLE
Localized strings from string-change tagged PRs

### PR DESCRIPTION
**Pull request by clementmouchet**
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [x] REQUIRED: Do you have an Issue **assigned to you** within the v1 milestone for this PR?  Put the Issue number here:
- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

After all boxes above are checked, request and receive an Approved review from any team member knowledgable in the area (TODO team member list).  Once approved, the team member will assign your review to a Committer or use the `needs-merge` label.

## What does this PR accomplish?

Extracted strings from https://github.com/WorldHealthOrganization/app/pulls?q=is%3Apr+label%3Astring-change+

## Did you add any dependencies?

No

### CODE REVIEW

File: client/flutter/lib/generated/intl/messages_en.dart [hunk 1]
```diff
@@ -49,13 +70,21 @@ class MessageLookup extends MessageLookupByLibrary {
...
     "locationSharingPageDescription" : MessageLookupByLibrary.simpleMessage("To get local news and information, allow location sharing."),
     "locationSharingPageTitle" : MessageLookupByLibrary.simpleMessage("Get the latest news from your community"),
     "newsFeedSliverListNewsFeedItemDescription1" : MessageLookupByLibrary.simpleMessage("Situation reports provide the latest updates on the novel coronavirus outbreak."),
     "newsFeedSliverListNewsFeedItemDescription2" : MessageLookupByLibrary.simpleMessage("Rolling updates on coronavirus disease (COVID-19) sourced from across WHO media."),
     "newsFeedSliverListNewsFeedItemDescription3" : MessageLookupByLibrary.simpleMessage("All news releases, statements, and notes for the media."),
-    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID-19) press briefings including videos, audio and transcripts."),
+    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID-2019) press briefings including videos, audio and transcripts."),
```
Comment (advayDev1) [inline client/flutter/lib/generated/intl/messages_en.dart]: COVID-19 is the official name we must use

File: client/flutter/lib/l10n/intl_en_US.arb [hunk 2]
```diff
@@ -3,16 +3,31 @@
...
+  "commonDialogButtonOk": "OK",
+  "commonContentLoadingDialogUpdateRequiredTitle": "App Update Required",
+  "commonContentLoadingDialogUpdateRequiredBodyText": "Please update to the latest version of the app in order to receive the latest information and updates.",
+  "@section": "Legal Landing Page",
+  "legalLandingPageTitle": "Official WHO COVID-19 Information App",
+  "legalLandingPageButtonGetStarted": "Get Started",
+  "legalLandingPageButtonAgree": "By proceeding, you agree to our",
+  "LegalLandingPageTermsOfServiceLinkText": "Terms of Service",
+  "legalLandingPageTermsOfServiceLinkUrl": "https://whocoronavirus.org/terms",
+  "legalLandingPageAnd": "and",
```
Comment (advayDev1) [inline client/flutter/lib/l10n/intl_en_US.arb]: must preserve the spaces here " and "

File: client/flutter/lib/l10n/intl_en_US.arb [hunk 3]
```diff
@@ -3,16 +3,31 @@
...
+  "commonWorldHealthOrganizationCoronavirusAppVersion": "Version {version} ({buildNumber})",
+  "commonWorldHealthOrganizationCoronavirusCopyright": "© {year} WHO",
   "commonPermissionRequestPageButtonSkip": "Skip",
+  "commonDialogButtonOk": "OK",
+  "commonContentLoadingDialogUpdateRequiredTitle": "App Update Required",
+  "commonContentLoadingDialogUpdateRequiredBodyText": "Please update to the latest version of the app in order to receive the latest information and updates.",
+  "@section": "Legal Landing Page",
+  "legalLandingPageTitle": "Official WHO COVID-19 Information App",
+  "legalLandingPageButtonGetStarted": "Get Started",
+  "legalLandingPageButtonAgree": "By proceeding, you agree to our",
```
Comment (advayDev1) [inline client/flutter/lib/l10n/intl_en_US.arb]: must preserve the newline
```suggestion
  "legalLandingPageButtonAgree": "By proceeding, you agree to our\n",
```

File: client/flutter/lib/l10n/intl_en_US.arb (hunk 2 above)
Comment (clementmouchet) [reply client/flutter/lib/l10n/intl_en_US.arb]: Ok, I can do but that's bad practice to rely on strings content to format the app. the right thing to do is to add padding around it, handling the style properly.

File: client/flutter/lib/l10n/intl_en_US.arb [hunk 4]
```diff
@@ -45,20 +60,17 @@
...
+  "travelAdvicePageListTitle": "Travellers returning from affected areas should:",
+  "travelAdvicePageListItem1Text": "Self-monitor for symptoms for 14 days and follow national protocols of receiving countries",
+  "travelAdvicePageListItem1Image": "assets/travel_advice/self-monitor.png",
+  "travelAdvicePageListItem2Text": "Some countries may require returning travellers to enter quarantine.",
+  "travelAdvicePageListItem2Image": "assets/travel_advice/quarantine.png",
+  "travelAdvicePageListItem3Text": "If symptoms occur, travellers are advsed to contact local health care providers, preferably by phone, and inform them of their symptoms and travel history.",
+  "travelAdvicePageListItem3Image": "assets/travel_advice/doctor.png",
+  "travelAdvicePageButtonGeneralRecommendations": "General\nRecommendations",
+  "travelAdvicePageButtonGeneralRecommendationsLink": "https://who.int",
```
Comment (advayDev1) [inline client/flutter/lib/l10n/intl_en_US.arb]: https://www.who.int/emergencies/diseases/novel-coronavirus-2019 everywhere

File: client/flutter/lib/l10n/intl_en_US.arb (hunk 3 above)
Comment (clementmouchet) [reply client/flutter/lib/l10n/intl_en_US.arb]: same as above, I can put it back but the more strings like that we have the more difficult it'll be to localize the app.

File: client/flutter/lib/l10n/intl_en_US.arb (hunk 4 above)
Comment (clementmouchet) [reply client/flutter/lib/l10n/intl_en_US.arb]: is it possible to treat the url fixes as a separate PR? all I did there was extract what's inline and cannot be translated, but needs to be. URLs won't be touched by translators and are only put there so we have the ability to change them to direct to pages localised in the right language.

File: client/flutter/lib/generated/intl/messages_en.dart (hunk 1 above)
Comment (clementmouchet) [reply client/flutter/lib/generated/intl/messages_en.dart]: done, thanks for spotting that

File: client/flutter/lib/l10n/intl_en_US.arb (hunk 4 above)
Comment (advayDev1) [reply client/flutter/lib/l10n/intl_en_US.arb]: SG!

File: client/flutter/lib/generated/intl/messages_en_GB.dart [hunk 5]
```diff
@@ -43,7 +43,7 @@ class MessageLookup extends MessageLookupByLibrary {
     "newsFeedSliverListNewsFeedItemDescription1" : MessageLookupByLibrary.simpleMessage("Situation reports provide the latest updates on the novel coronavirus outbreak."),
     "newsFeedSliverListNewsFeedItemDescription2" : MessageLookupByLibrary.simpleMessage("Rolling updates on coronavirus disease (COVID-19) sourced from across WHO media."),
     "newsFeedSliverListNewsFeedItemDescription3" : MessageLookupByLibrary.simpleMessage("All news releases, statements, and notes for the media."),
-    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID-19) press briefings including videos, audio and transcripts."),
+    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID-2019) press briefings including videos, audio and transcripts."),
```
Comment (advayDev1) [inline client/flutter/lib/generated/intl/messages_en_GB.dart]:
```suggestion
    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID-19) press briefings including videos, audio and transcripts."),
```

Comment (clementmouchet) [reply client/flutter/lib/generated/intl/messages_en_GB.dart]: I've not made any updates to the translated files, you can ignore then, they'll be deleted or replaced before the release

**Comment by clementmouchet** [comment]
thanks @advayDev1 !

**Pull request merged**